ACKNOWLEDGMENTS.md

```diff
@@ -7,6 +7,7 @@ with a short description of your contribution(s) below. For example:
 
 MLX was developed with contributions from the following individuals:
 
+- Manish Aradwad: Added support for deepcopy.
```

Comment:
Could you make this slightly more descriptive: Added support to pickle `nn.Module`?

Reply:
Sure!